scripts/models/token/token.constants.js

```diff
@@ -25,6 +25,7 @@ const SERIAL_SPECIFICATIONS = {
 const TOKEN_FORMATS = {
   STANDARD: "standard",
   AUTOMATED: "automated",
+  DESIGN: "design",
```

Comment:
can we somewhere document the different token formats? for me its not 100% clear whats the exact difference between those 3. thanks